controllers/api/v1alpha1/appworkload_types.go

```diff
@@ -69,6 +69,8 @@ type AppWorkloadStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
Adding this annotation seem to remove [the appworload status](https://github.com/cloudfoundry/korifi/blob/724bf33dce620ee1eb3ded5882aef4bbaaae571c/helm/korifi/controllers/crds/korifi.cloudfoundry.org_appworkloads.yaml#L806-L807) from the generated manifest and this is causing controllers test to [fail](https://github.com/cloudfoundry/korifi/actions/runs/11572989343/job/32214943264) as the app workload resource has no `Status` anymore, while the test wants to update it.

In order to reproduce locally, run `make generate manifests` and then run the process controller tests: `scripts/test ./controllers/controllers/workloads/processes`

I cannot explain why this occurs